Jenkinsfile

```diff
@@ -951,8 +987,162 @@ pipeline {
+                                           clush -B -S -o '-i ci_key' -l jenkins -w ${tnodes} \
+                                               "sudo mkdir -p $DAOS_BASE;sudo mount -t nfs $HOSTNAME:$PWD $DAOS_BASE"
+
+                                           clush -B -S -o '-i ci_key' -l jenkins -w ${tnodes} \
+                                             "if [ ! -e /dev/pmem0 ]; then sudo ${DAOS_BASE}/install/bin/daos_server storage prepare --scm-only --force; fi"''',
+                                failure_artifacts: env.STAGE_NAME
+                        provisionNodes NODELIST: env.NODELIST,
+                                       node_count: 4,
+                                       power_only: true,
+                                       inst_repos: el7_daos_repos,
+                                       inst_rpms: 'daos-' + daos_packages_version +
+                                                  ' daos-client-' + daos_packages_version +
+                                                  ' cart-' + env.CART_COMMIT + ' ' +
+                                                  el7_functional_rpms
+                        runTest stashes: [ 'CentOS-install', 'CentOS-build-vars' ],
+                                script: '''tnodes=$(echo $NODELIST | cut -d ',' -f 1-9)
+                                           # set DAOS_TARGET_OVERSUBSCRIBE env here
+                                           export DAOS_TARGET_OVERSUBSCRIBE=1
+                                           . ./.build_vars.sh
+                                           DAOS_BASE=${SL_PREFIX%/install*}
+                                           clush -B -S -o '-i ci_key' -l jenkins -w ${tnodes} \
+                                               "sudo mkdir -p $DAOS_BASE;sudo mount -t nfs $HOSTNAME:$PWD $DAOS_BASE"
+                                           clush -B -S -o '-i ci_key' -l jenkins -w ${tnodes} \
+                                             "if [ ! -e /dev/pmem0 ]; then sudo ${DAOS_BASE}install/bin/daos_server storage prepare --scm-only --force; fi"'''
+                    }
+                    //post {
+                    //    always {
+                    //        // Dismount is currently failing.
+                    //        sh '''. ./.build_vars.sh
+                    //              DAOS_BASE=${SL_PREFIX%/install*}
+                    //              NODE=${NODELIST%%,*}
+                    //              . ./.build_vars.sh
+                    //              DAOS_BASE=${SL_PREFIX%/install*}
+                    //              clush -B -S -o '-i ci_key' -l jenkins -w ${tnodes} \
+                    //                "if ! sudo umount ${DAOS_BASE}; then echo \"Failed to unmount $DAOS_BASE\"; ps axf; fi"
+                    //              '''
+                    //    }
+                    //}
+                } // stage('Functional_Hardware_setup 1') {
+                stage('Functional_Hardware_setup 2') {
```

Comment:
Is this supposed to be running on the same set of nodes as the previous stage?  In your most recent Jenkins run, you got a different set of nodes from the above stage.